### ISSUE TITLE
Organize web files

### ISSUE THREAD

**Issue by amandavisconti** [scholarslab/scholarslab.org#49]
Organize the website file folder so it's easier to find things (e.g. all static pages in one folder)

**Comment by amandavisconti** [comment]
<img width="644" alt="screen shot 2018-01-11 at 11 10 40 am" src="https://user-images.githubusercontent.com/899033/34836440-e5eb0894-f6c5-11e7-8b68-f13f00702f8f.png">
Moved some things around—mostly static pages into /_pages

**Comment by amandavisconti** [comment]
Need to get redirects working

**Comment by amandavisconti** [comment]
Tidied pages into /pages:
- Added permalinks in each page’s YAML to preserve URL structure
- Redirects /archives to /blog
<img width="400" alt="screen shot 2018-01-18 at 12 13 39 pm" src="https://user-images.githubusercontent.com/899033/35111334-0951d122-fc49-11e7-9106-60be966a1757.png">

This works locally, but I haven't pushed it up because rake is breaking on some html-proofer errors, e.g.

- ./_site/announcements/call-for-spring-2018-makerspace-technologist-applications/index.html
  *  internally linking to /makerspace, which does not exist (line 98)
     <a href="/makerspace">Makerspace</a>

But /makerspace does exist and work fine on my local site?

**Comment by walshbr** [comment]
I went ahead and did the grad fellowship content on its own branch, and I ran into the same html-proofer errors for /graduate-fellowships, which exists as well. 

Not sure why that is a problem, but I fixed it by passing into the absolute URL rather than the absolute. I only had four instances, so it wasn't too painful to go through the posts and fix four instances of /graduate-fellowships to http://scholarslab.org/graduate-fellowships/. Might work for you as well.
